### PR TITLE
Rework of #9266 - Refactor Carbanak related rules filenames to use data source

### PR DESCRIPTION
**Pull request by Spades0**
## Description
|Related PR|Related Issue|Closes Issue|
|---|---|---|
| [#9271](https://github.com/wazuh/wazuh/pull/9271) | [#9266](https://github.com/wazuh/wazuh/issues/9266)   |[#11283](https://github.com/wazuh/wazuh/issues/11283)|

## Changes & Comments
The aim of this PR is:

Add copyright headers,
Fix tag ordering and alphabetical order in group tag.
To ensure the rules and decoders from issue [#11283] work correctly.
To fix issues regarding indentation, syntax, rule granularity, spellings and ensure the ruleset quality is standard.
Note that .ini fules are commented because windows event channel testing is not supported by run_tests.py yet.
Rule 92065 is wrongly, included in sysmon 1 rules, moved to windows event channel as 92265.

### Syntax

- [x] 1.a Rule tags order must be compliant.
- [x] 1.b Decoder tags order must be compliant.
- [x] 1.c XML blocks must as compact as possible.
- [x] 1.d Only one empty line between rule/decoder and the next rule/decoder.
- [x] 1.e Decoder extracted fields names must use '_' whether a space is needed.
- [x] 1.f Decoder name must use '-' whether a space is needed.
- [x] 2.d XML and ini files must use correct indentation
 
### Grammar

- [x] 2.a Grammar quality.
- [x] 2.b Similar phrases must keep tenses and expressions.
- [x] 2.c Grammar basic rules like capitalization, punctuation marks, sentence construction, and others.

### Semantic

- [x] 3.a New decoder, rule, or test are written in the correct file and grouped correctly inside the file.
- [x] 3.b Group similar rules under same ID's group.
- [x] 3.c: 
   - [x] 3.c1 Find and reuse group name before creating a new one.
   - [x] 3.c2 Include a new group definition in a PR comment.
   - [x] 3.c3 Any group name must use '_' whether it does need a space char.
   - [x] 3.c4 The groups inside a tag must be sorted in alphabetical order.
   - [x] 3.d Rule level should be compliant with the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html).

### Unit testing

- [x] 4.a A metadata block at the beginning of the .ini file describing software, version, and logs source.
- [x] 4.b Each new rule must have at least one test entry in the correct .ini file.
- [x] 4.c Runtest.py must pass and must include the results in raw format here.

**Unit Tests**
<details>
  <statement></statement>

  ```
Restarting wazuh-manager...
|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    0     |   4174   |  0.00%   |
| Decoders |    0     |   172    |  0.00%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
Restarting wazuh-manager...
root@ubuntu-focal:/var/ossec/ruleset/feed/ruleset/testing# python3 runtests.py 
Restarting wazuh-manager...
- [ File = ./tests/samba.ini ] ---------
....

- [ File = ./tests/fortigate.ini ] ---------
........................................

- [ File = ./tests/sysmon_eid_7.ini ] ---------
...

- [ File = ./tests/apparmor.ini ] ---------
.....

- [ File = ./tests/pam.ini ] ---------
.....

- [ File = ./tests/audit_lateral.ini ] ---------
......

- [ File = ./tests/systemd.ini ] ---------
..

- [ File = ./tests/arbor.ini ] ---------
..

- [ File = ./tests/dovecot.ini ] ---------
...............

- [ File = ./tests/mailscanner.ini ] ---------
.

- [ File = ./tests/ossec.ini ] ---------
.....

- [ File = ./tests/vuln_detector.ini ] ---------
..

- [ File = ./tests/squid_rules.ini ] ---------
..

- [ File = ./tests/pix.ini ] ---------
......................

- [ File = ./tests/huawei_usg.ini ] ---------
...

- [ File = ./tests/checkpoint_smart1.ini ] ---------
..................

- [ File = ./tests/junos.ini ] ---------
...

- [ File = ./tests/web_appsec.ini ] ---------
...............................

- [ File = ./tests/cimserver.ini ] ---------
..

- [ File = ./tests/cisco_ftd.ini ] ---------
..........................................

- [ File = ./tests/firewalld.ini ] ---------
..

- [ File = ./tests/audit_recon.ini ] ---------
..

- [ File = ./tests/openldap.ini ] ---------
.........

- [ File = ./tests/sophos_fw.ini ] ---------
..........

- [ File = ./tests/owlh.ini ] ---------
....

- [ File = ./tests/freepbx.ini ] ---------
......

- [ File = ./tests/syslog.ini ] ---------
......

- [ File = ./tests/openvpn_ldap.ini ] ---------
..

- [ File = ./tests/sophos.ini ] ---------
........

- [ File = ./tests/unbound.ini ] ---------


- [ File = ./tests/sysmon_eid_3.ini ] ---------
......

- [ File = ./tests/github.ini ] ---------
....................................................................................................................................................................................................................................................................................................................................

- [ File = ./tests/cpanel.ini ] ---------
.......

- [ File = ./tests/fireeye.ini ] ---------
...

- [ File = ./tests/fortiauth.ini ] ---------
....

- [ File = ./tests/iptables.ini ] ---------
........

- [ File = ./tests/named.ini ] ---------
.....

- [ File = ./tests/sophos-utm-firewall.ini ] ---------
......

- [ File = ./tests/nginx.ini ] ---------
............

- [ File = ./tests/exim.ini ] ---------
.......

- [ File = ./tests/opensmtpd.ini ] ---------
.......

- [ File = ./tests/oscap.ini ] ---------
................................

- [ File = ./tests/gitlab.ini ] ---------
.........................

- [ File = ./tests/paloalto.ini ] ---------
................

- [ File = ./tests/api.ini ] ---------
............................

- [ File = ./tests/aws_s3_access.ini ] ---------
.......

- [ File = ./tests/exchange.ini ] ---------
..

- [ File = ./tests/f5_big_ip.ini ] ---------
...........................................

- [ File = ./tests/cisco_asa.ini ] ---------
........................................................................................

- [ File = ./tests/nextcloud.ini ] ---------
.......

- [ File = ./tests/trendmicro-cloud-one.ini ] ---------
.................

- [ File = ./tests/pfsense.ini ] ---------
..

- [ File = ./tests/vsftpd.ini ] ---------
....

- [ File = ./tests/web_rules.ini ] ---------
..........

- [ File = ./tests/apache.ini ] ---------
..................

- [ File = ./tests/eset.ini ] ---------
........

- [ File = ./tests/doas.ini ] ---------
....

- [ File = ./tests/sysmon.ini ] ---------
...

- [ File = ./tests/gcp.ini ] ---------
...........

- [ File = ./tests/icinga.ini ] ---------
....

- [ File = ./tests/postfix.ini ] ---------
..

- [ File = ./tests/audit_scp.ini ] ---------
.

- [ File = ./tests/dropbear.ini ] ---------
...

- [ File = ./tests/rsh.ini ] ---------
..

- [ File = ./tests/kernel_usb.ini ] ---------
......

- [ File = ./tests/mcafee_epo.ini ] ---------
.

- [ File = ./tests/netscreen.ini ] ---------
....

- [ File = ./tests/win_event_channel.ini ] ---------
......

- [ File = ./tests/panda_paps.ini ] ---------
........

- [ File = ./tests/powershell.ini ] ---------
...

- [ File = ./tests/sysmon_eid_1.ini ] ---------
..........................

- [ File = ./tests/auditd.ini ] ---------
...

- [ File = ./tests/sysmon_eid_11.ini ] ---------
.......

- [ File = ./tests/cylance.ini ] ---------
.......

- [ File = ./tests/modsecurity.ini ] ---------
......

- [ File = ./tests/cloudlfare-waf.ini ] ---------
.............

- [ File = ./tests/office365.ini ] ---------
................................................................................................................................

- [ File = ./tests/sshd.ini ] ---------
...........................................

- [ File = ./tests/cisco_ios.ini ] ---------
.................

- [ File = ./tests/glpi.ini ] ---------
...

- [ File = ./tests/sudo.ini ] ---------
........

- [ File = ./tests/su.ini ] ---------
.....

- [ File = ./tests/proftpd.ini ] ---------
.......

- [ File = ./tests/SonicWall.ini ] ---------
........

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |   1032   |   4174   |  24.72%  |
| Decoders |    76    |   172    |  44.19%  |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/samba.ini        |    4     |    0     |    ✅     |
|./tests/fortigate.ini    |    40    |    0     |    ✅     |
|./tests/sysmon_eid_7.ini |    3     |    0     |    ✅     |
|./tests/apparmor.ini     |    5     |    0     |    ✅     |
|./tests/pam.ini          |    5     |    0     |    ✅     |
|./tests/audit_lateral.ini|    6     |    0     |    ✅     |
|./tests/systemd.ini      |    2     |    0     |    ✅     |
|./tests/arbor.ini        |    2     |    0     |    ✅     |
|./tests/dovecot.ini      |    15    |    0     |    ✅     |
|./tests/mailscanner.ini  |    1     |    0     |    ✅     |
|./tests/ossec.ini        |    5     |    0     |    ✅     |
|./tests/vuln_detector.ini|    2     |    0     |    ✅     |
|./tests/squid_rules.ini  |    2     |    0     |    ✅     |
|./tests/pix.ini          |    22    |    0     |    ✅     |
|./tests/huawei_usg.ini   |    3     |    0     |    ✅     |
|./tests/checkpoint_smart1.ini|    18    |    0     |    ✅     |
|./tests/junos.ini        |    3     |    0     |    ✅     |
|./tests/web_appsec.ini   |    31    |    0     |    ✅     |
|./tests/cimserver.ini    |    2     |    0     |    ✅     |
|./tests/cisco_ftd.ini    |    42    |    0     |    ✅     |
|./tests/firewalld.ini    |    2     |    0     |    ✅     |
|./tests/audit_recon.ini  |    2     |    0     |    ✅     |
|./tests/openldap.ini     |    9     |    0     |    ✅     |
|./tests/sophos_fw.ini    |    10    |    0     |    ✅     |
|./tests/owlh.ini         |    4     |    0     |    ✅     |
|./tests/freepbx.ini      |    6     |    0     |    ✅     |
|./tests/syslog.ini       |    6     |    0     |    ✅     |
|./tests/openvpn_ldap.ini |    2     |    0     |    ✅     |
|./tests/sophos.ini       |    8     |    0     |    ✅     |
|./tests/unbound.ini      |    0     |    0     |    ✅     |
|./tests/sysmon_eid_3.ini |    6     |    0     |    ✅     |
|./tests/github.ini       |   324    |    0     |    ✅     |
|./tests/cpanel.ini       |    7     |    0     |    ✅     |
|./tests/fireeye.ini      |    3     |    0     |    ✅     |
|./tests/fortiauth.ini    |    4     |    0     |    ✅     |
|./tests/iptables.ini     |    8     |    0     |    ✅     |
|./tests/named.ini        |    5     |    0     |    ✅     |
|./tests/sophos-utm-firewall.ini|    6     |    0     |    ✅     |
|./tests/nginx.ini        |    12    |    0     |    ✅     |
|./tests/exim.ini         |    7     |    0     |    ✅     |
|./tests/opensmtpd.ini    |    7     |    0     |    ✅     |
|./tests/oscap.ini        |    32    |    0     |    ✅     |
|./tests/gitlab.ini       |    25    |    0     |    ✅     |
|./tests/paloalto.ini     |    16    |    0     |    ✅     |
|./tests/api.ini          |    28    |    0     |    ✅     |
|./tests/aws_s3_access.ini|    7     |    0     |    ✅     |
|./tests/exchange.ini     |    2     |    0     |    ✅     |
|./tests/f5_big_ip.ini    |    43    |    0     |    ✅     |
|./tests/cisco_asa.ini    |    88    |    0     |    ✅     |
|./tests/nextcloud.ini    |    7     |    0     |    ✅     |
|./tests/trendmicro-cloud-one.ini|    17    |    0     |    ✅     |
|./tests/pfsense.ini      |    2     |    0     |    ✅     |
|./tests/vsftpd.ini       |    4     |    0     |    ✅     |
|./tests/web_rules.ini    |    10    |    0     |    ✅     |
|./tests/apache.ini       |    18    |    0     |    ✅     |
|./tests/eset.ini         |    8     |    0     |    ✅     |
|./tests/doas.ini         |    4     |    0     |    ✅     |
|./tests/sysmon.ini       |    3     |    0     |    ✅     |
|./tests/gcp.ini          |    11    |    0     |    ✅     |
|./tests/icinga.ini       |    4     |    0     |    ✅     |
|./tests/postfix.ini      |    2     |    0     |    ✅     |
|./tests/audit_scp.ini    |    1     |    0     |    ✅     |
|./tests/dropbear.ini     |    3     |    0     |    ✅     |
|./tests/rsh.ini          |    2     |    0     |    ✅     |
|./tests/kernel_usb.ini   |    6     |    0     |    ✅     |
|./tests/mcafee_epo.ini   |    1     |    0     |    ✅     |
|./tests/netscreen.ini    |    4     |    0     |    ✅     |
|./tests/win_event_channel.ini|    6     |    0     |    ✅     |
|./tests/panda_paps.ini   |    8     |    0     |    ✅     |
|./tests/powershell.ini   |    3     |    0     |    ✅     |
|./tests/sysmon_eid_1.ini |    26    |    0     |    ✅     |
|./tests/auditd.ini       |    3     |    0     |    ✅     |
|./tests/sysmon_eid_11.ini|    7     |    0     |    ✅     |
|./tests/cylance.ini      |    7     |    0     |    ✅     |
|./tests/modsecurity.ini  |    6     |    0     |    ✅     |
|./tests/cloudlfare-waf.ini|    13    |    0     |    ✅     |
|./tests/office365.ini    |   128    |    0     |    ✅     |
|./tests/sshd.ini         |    43    |    0     |    ✅     |
|./tests/cisco_ios.ini    |    17    |    0     |    ✅     |
|./tests/glpi.ini         |    3     |    0     |    ✅     |
|./tests/sudo.ini         |    8     |    0     |    ✅     |
|./tests/su.ini           |    5     |    0     |    ✅     |
|./tests/proftpd.ini      |    7     |    0     |    ✅     |
|./tests/SonicWall.ini    |    8     |    0     |    ✅     |
Restarting wazuh-manager...
  ```
</details>

**Individual Tests**
<details>
<statement></statement>

```
Restarting wazuh-manager...
- [ File = ./tests/sysmon_eid_1.ini ] ---------
..........................

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    26    |   4174   |  0.62%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/sysmon_eid_1.ini |    26    |    0     |    ✅     |
Restarting wazuh-manager...

Restarting wazuh-manager...
- [ File = ./tests/sysmon_eid_3.ini ] ---------
......

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    6     |   4174   |  0.14%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/sysmon_eid_3.ini |    6     |    0     |    ✅     |
Restarting wazuh-manager...

Restarting wazuh-manager...
- [ File = ./tests/sysmon_eid_7.ini ] ---------
...

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    3     |   4174   |  0.07%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/sysmon_eid_7.ini |    3     |    0     |    ✅     |
Restarting wazuh-manager...

Restarting wazuh-manager...
- [ File = ./tests/sysmon_eid_11.ini ] ---------
.......

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    7     |   4174   |  0.17%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/sysmon_eid_11.ini|    7     |    0     |    ✅     |
Restarting wazuh-manager...

Restarting wazuh-manager...
- [ File = ./tests/win_event_channel.ini ] ---------
......

|Component |  Tested  |  Total   | Coverage |
| -------- | -------- | -------- | -------- |
|  Rules   |    5     |   4174   |  0.12%   |
| Decoders |    1     |   172    |  0.58%   |

|          File           |  Passed  |  Failed  |  Status  |
|        --------         | -------- | -------- | -------- |
|./tests/win_event_channel.ini|    6     |    0     |    ✅     |
Restarting wazuh-manager...
```
</details>

- [x] 4.d New CDB lists must include the proper test entry in the correct .ini file.

### E2E testing

- [x] 5.a Logs for new or modified decoder/rules sent to a manager running with this PR ruleset appear in Kibana. 

image
image
image

- [x] 5.b There are not affected or broken visualizations on Kibana.

image

- [x] 5.c New or modified items can be seen correctly using APP ruleset navigation.

image
image
image
image
image

### Elasticsearch Template

- [x] 6.a Known fields with output format managed and usually used for searching are included in template array index.query.default_field. 
- [x] 6.b The new field with the correct date format is stored as a "date" type field in the template.
- [x] 6.c The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template. 
- [x] 6.d Known fields with output format managed and usually used for searching are included in the template.

### Stoppers

- [x] 7.a No previous rule ID changes without triple check.
- [x] 7.b No previous decoder name changes without triple check.
- [x] 7.c No previous file name changes without triple check.
- [x] 7.d No previous test changes its 'rule' field value without triple check.

### Others

- [x] 8.a Each file has the correct copyright block.
- [x] 8.b The copyright block doesn't have "Author" only "Created by Wazuh". To include an "Author" request triple check.
- [x] 8.c The copyright block doesn't use "-". 
- [x] 8.d The rule files don't have any sample log.
- [x] 8.e The decoder file has sample logs next to the decoder that matches that log.
- [x] 8.f The decoder and rule files have information about software, version, and any helpful information.
- [x] 8.g The PR has a single commit with CHANGELOG changes in the correct format.
- [x] 8.h The new rule ID is not in use.
- [x] 8.i The new rule ID must be in the defined IDs range. 
- [x] 8.j New rules ID range must be verified with a triple check and noted in the rules ID document.
- [x] 8.k The new extracted IP fields are in the pipeline as "geo" field and "geo_point" type in the template.

### CODE REVIEW

**Review by fabamatic — CHANGES_REQUESTED** [review]
Lets use sysmon 7, 11 and windows event channel rule files from this PR: https://github.com/wazuh/wazuh/pull/11327 (as that version has corrected rule ids for those files).
Revert:
ruleset/rules/0830-sysmon_id_11.xml
ruleset/rules/0820-sysmon_id_7.xml
ruleset/rules/0840-win_event_channel.xml

Remove:
ruleset/testing/tests/sysmon_eid_7.ini
ruleset/testing/tests/win_event_channel.ini
ruleset/testing/tests/sysmon_eid_11.ini

File: ruleset/rules/0800-sysmon_id_1.xml [hunk 1]
```diff
@@ -1,300 +1,271 @@
...
-        <field name="win.eventdata.originalFileName" type="pcre2">(?i)reg.EXE</field>
-        <field name="win.eventdata.commandLine" type="pcre2">(?i)save.+\\\\SAM</field>
-        <description>Reg.exe used to dump SAM hive</description>
-        <mitre>
-            <id>T1003.002</id>
-        </mitre>
-    </rule>
+<!-- 
+  Sysmon Event ID 1 rules           
+-->
```
Comment (jcruzlp) [inline ruleset/rules/0800-sysmon_id_1.xml:8]: We should add the ID range

File: ruleset/rules/0800-sysmon_id_1.xml [hunk 2]
```diff
@@ -1,300 +1,271 @@
...
+  <rule id="92013" level="6">
+    <if_sid>92010</if_sid>
+    <field name="win.eventdata.image" type="pcre2">(?i)[c-z]:(\\\\Users\\\\Public\\\\|\\\\Users\\\\.+\\\\AppData\\\\local\\\\temp)</field>
+    <description>Scripting interpreter spawned a process from a suspicious path.</description>
+    <mitre>
+      <id>T1059</id>
+    </mitre>
+  </rule>
+
+  <rule id="92021" level="4">
```
Comment (jcruzlp) [inline ruleset/rules/0800-sysmon_id_1.xml:47]: If we need to have those ID jumps, we should justify them.

File: ruleset/rules/0810-sysmon_id_3.xml [hunk 3]
```diff
@@ -1,74 +1,70 @@
...
+    <if_group>sysmon_event3</if_group>
+    <field name="win.eventdata.image" type="pcre2">(?i)\\powershell\.exe</field>
+    <field name="win.eventdata.destinationPort">^135$</field>
+    <description>Suspicious DCOM/RPC activity from Powershell process.</description>
+    <mitre>
+      <id>T1110</id>
+    </mitre>
+  </rule>
+
+  <rule id="92160" level="6">
```
Comment (jcruzlp) [inline ruleset/rules/0810-sysmon_id_3.xml:21]: Ditto here

**Review by jcruzlp — CHANGES_REQUESTED** [review]
Check ID ranges and ID jumps (If they are mandatory, we should justify them)

File: ruleset/rules/0800-sysmon_id_1.xml (hunk 1 above)
Comment (fabamatic) [reply ruleset/rules/0800-sysmon_id_1.xml:8]: Done

File: ruleset/rules/0800-sysmon_id_1.xml (hunk 2 above)
Comment (fabamatic) [reply ruleset/rules/0800-sysmon_id_1.xml]: Done

File: ruleset/rules/0810-sysmon_id_3.xml (hunk 3 above)
Comment (fabamatic) [reply ruleset/rules/0810-sysmon_id_3.xml]: Done

**Review by fabamatic — APPROVED** [review]

File: ruleset/rules/0800-sysmon_id_1.xml [hunk 4]
```diff
@@ -1,300 +1,261 @@
...
+
+  <rule id="92021" level="0">
+    <if_group>sysmon_event1</if_group>
+    <field name="win.eventdata.originalFileName" type="pcre2">(?i)PowerShell\.EXE</field>
+    <field name="win.eventdata.commandLine" type="pcre2">(?i)(Remove-Item|ri|rm|rmdir|del|erase|rd)\b</field>
+    <description>Powershell was used to delete files or directories.</description>
+    <mitre>
+      <id>T107.004</id>
+    </mitre>
+  </rule>
```
Comment (nmkoremblum) [inline ruleset/rules/0800-sysmon_id_1.xml:218]: Is this OK to be a level 0 alert?

**Review by nmkoremblum — CHANGES_REQUESTED** [review]

File: ruleset/rules/0840-win_event_channel.xml [hunk 5]
```diff
@@ -1,79 +1,84 @@
...
-    <rule id="92036" level="6">
-        <if_sid>92035</if_sid>
-        <field name="win.eventdata.authenticationPackageName" type="pcre2">NTLM</field>
-        <description>Successful Remote Logon Detected - NTLM authentication, possible pass-the-hash attack</description>
-        <mitre>
-            <id>T1550.002</id>
-        </mitre>
-        <group>authentication_success,pci_dss_10.2.5,gpg13_7.1,gpg13_7.2,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
-    </rule>
+  <rule id="92653" level="0">
```
Comment (juliancnn) [inline ruleset/rules/0840-win_event_channel.xml:43]: According to the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html), a successful login should be level 3.
```suggestion
  <rule id="92653" level="3">
```

Comment (juliancnn) [inline ruleset/rules/0840-win_event_channel.xml:43]: According to the [documentation](https://documentation.wazuh.com/current/user-manual/ruleset/rules-classification.html), a successful login should be level 3.
```suggestion
  <rule id="92653" level="3">
```

**Review by juliancnn — CHANGES_REQUESTED** [review]

File: ruleset/rules/0840-win_event_channel.xml (hunk 5 above)
Comment (fabamatic) [reply ruleset/rules/0840-win_event_channel.xml]: Agree

File: ruleset/rules/0800-sysmon_id_1.xml (hunk 4 above)
Comment (fabamatic) [reply ruleset/rules/0800-sysmon_id_1.xml:218]: rising level to 3

File: ruleset/testing/tests/audit_scp.ini [hunk 6]
```diff
@@ -1,5 +1,42 @@
...
+rule = 92605
 alert = 6
 decoder = auditd
```
Comment (nmkoremblum) [inline ruleset/testing/tests/audit_scp.ini:42]:
```suggestion
rule = 92605
alert = 6
decoder = auditd

```

**Review by nmkoremblum — CHANGES_REQUESTED** [review]
Newline at the end is missing

**Review by 72nomada — APPROVED** [review]
LGTM

**Review by juliancnn — APPROVED** [review]
LGTM!

**Review by nmkoremblum — APPROVED** [review]
LGTM

**Pull request merged**